README.md

```diff
@@ -48,7 +48,7 @@ The Resolver will properly follow CNAME chains as well as SRV record lookups. Th
 
 ## Client
 
-The Hickory DNS Client is intended to be used for operating against a DNS server directly. It can be used for verifying records or updating records for servers that support SIG0 and dynamic update. The Client is also capable of validating DNSSEC. As of now NSEC3 validation is not yet supported, though NSEC is. There are two interfaces that can be used, the async/await compatible AsyncClient and a blocking Client for ease of use. Today, Tokio is required for the executor Runtime.
+The Hickory DNS Client is intended to be used for operating against a DNS server directly. It can be used for verifying records or updating records for servers that support SIG0 and dynamic update. The Client is also capable of validating DNSSEC. As of now NSEC3 validation is not yet supported, though NSEC is. Today, the Tokio async runtime is required.
```

Comment:
Different issue, but per my understanding we now have some support for NSEC3 validation?